File: sem 4/Data Structure/UNIT1/qu2.cpp
#include<iostream>

using namespace std;

void display(int arr[],int n)
{
    cout<<"Array elements : "<<endl;

    for(int i=0;i<n;i++)
    {
        cout<<arr[i]<<"\t";
    }
}
int main()
{
    int n;

    cout<<"Enter the size of array : ";
    cin>>n;

    int arr[n];

    cout<<"Enter the array elements : ";
    for(int i=0;i<n;i++)
    {
        cin>>arr[i];
    }

    display(arr,n);
}

File: sem 4/JAVA/UNIT_2/Qu2.java
import java.util.*;

class Qu2 
{
    public static void main(String[] args) 
    {
        int arr[]=new int[5];

        System.out.println("Enter the array elements : ");
        Scanner sc=new Scanner(System.in);

        for(int i=0;i<arr.length;i++)
        {
            arr[i]=sc.nextInt();
        }

        Set<Integer>set=new HashSet<>();

        //If number is already in the set HashSet will skip 
        for(int num:arr)
        {
            set.add(num);
        }

        System.out.println("Remove Duplicate elements : ");
        for(int num:set)
        {
            System.out.print(num+" ");
        }
    }
}


//Set<Integer>set=new Set<>();
//it is not work because Set is just the rulebook.
//You cannot make an object from a rulebook.
//Set like a rulebook and HashSet is the implementation like an actual class following the rulebook
